composer.json

```
@@ -5,9 +5,9 @@
     "type": "library",
     "require": {
         "php": ">=5.3.7",
-        "symfony/dependency-injection": "^2.3.0|^3|^4|^5",
-        "symfony/filesystem": "^2.3.0|^3|^4|^5",
-        "symfony/config": "^2.3.0|^3|^4|^5"
+        "symfony/dependency-injection": "^2.3.0|^3|^4|^5|^6.0",
+        "symfony/filesystem": "^2.3.0|^3|^4|^5|^6.0",
+        "symfony/config": "^2.3.0|^3|^4|^5|^6.0"
```

```suggestion
        "symfony/dependency-injection": "^2.3|^3|^4|^5|^6",
        "symfony/filesystem": "^2.3|^3|^4|^5|^6",
        "symfony/config": "^2.3|^3|^4|^5|^6"
```
We can remove zeros for consistency.